app/src/main/java/org/jellyfin/mobile/webapp/RemoteVolumeProvider.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.webapp
+
+import android.media.AudioManager
+import android.media.VolumeProvider
+import org.jellyfin.mobile.utils.Constants
+
+class RemoteVolumeProvider(private val webappFunctionChannel: WebappFunctionChannel) : VolumeProvider(VOLUME_CONTROL_ABSOLUTE, 100, 0) {
+    override fun onAdjustVolume(direction: Int) {
+        when (direction) {
+            AudioManager.ADJUST_RAISE -> {
+                webappFunctionChannel.triggerInputManagerAction(Constants.INPUT_MANAGER_COMMAND_VOL_UP)
```

Comment:
Fix your TODO's

Reply:
They were already there in master xD

Reply:
Is it something we can fix or is it not possible without changing webui?

Reply:
It needs changes to the web UI, unfortunately. But then again, we'll probably have a native UI before we get those changes, lol.